mk/spksrc.tc-vers.mk

```diff
@@ -1,61 +1,65 @@
 ifeq ($(strip $(TC_OS_MIN_VER)),)
-TC_OS_MIN_VER = $(TC_FIRMWARE)
+TC_OS_MIN_VER = $(TC_OS_MIN_VER)
 endif
 
```

Comment:
This is more or less useless i.e. it does nothing (I read: if TC_OS_MIN_VER is empyt assign it to itself)

As this is already merged: keep in mind for a future update....

Reply:
This was me being too quick when pushing in the change...
Fixed right after with this commit already in https://github.com/SynoCommunity/spksrc/pull/4200/commits/3ffb64ef9a4fffc300bbb2fb3448a0039e8df988